pkg/tfbridge/internal/schemaconvert/schemaconvert.go

```diff
@@ -0,0 +1,134 @@
+package schemaconvert
```

Comment:
We need to add a (license) and a note explaining that this is test code, which should not be used in production parts of the bridge. Even better if this could live in `internal/test/schemaconvert`.